docs/docs/icicle/primitives/poseidon.md

```diff
@@ -16,7 +16,9 @@ Poseidon starts with the initialization of its internal state, which is composed
 
 This is done to prevent collisions and to prevent certain cryptographic attacks by ensuring that the internal state is sufficiently mixed and unpredictable.
 
-![Alt text](image.png)
+![First Board - 1redone](https://github.com/ingonyama-zk/icicle/assets/122266060/52257f5d-6097-47c4-8f17-7b6449b9d162)
```

Comment:
The "Alt text" portion should be a description of what the image conveys.

Reply:
That info is missing in the original as far as i can see